bookstore/src/components/Header.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Link } from 'react-router-dom';
+
+function Header() {
+    return (
+        <>
+        <Link className="nav" to="/BooksList">Books</Link>
```

Comment:
- [ ] As the route for rendering the `BookList` was changed, kindly make sure this link also links to the root of the project(`"/"`).